packages/gestalt-codemods/0.108.0-0.109.0/__tests__/convert-font-weight.test.js

```diff
@@ -1,7 +1,7 @@
 import { defineTest } from 'jscodeshift/dist/testUtils.js';
 
 jest.mock('../convert-font-weight', () => {
-  return Object.assign(require.requireActual('../convert-font-weight'), {
+  return Object.assign(jest.requireActual('../convert-font-weight'), {
```

Comment:
`require.requireActual` got removed in Jest v26: https://github.com/facebook/jest/releases/tag/v26.0.0
Related PR: https://github.com/facebook/jest/pull/9854